### PR TITLE
change peers into ts

### PR DESCRIPTION
**Pull request by zhouyuhe**
Add types to peers components
Add test to PeerReducer

### CODE REVIEW

File: src/features/peers/redux/actions.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { UPDATE_TOP_PEERS } from "./constants";
+import { ActionWithPayload } from "../../../store/actions";
+
+export type PeersData = {
+  symbol: string;
+  name: string;
```
Comment (danielabbasi) [inline src/features/peers/redux/actions.ts:6]: Q: does your peers data include names and symbols or just symbols?

File: src/features/peers/components/Peers.tsx [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
 import { updateStockAction } from "../../../actions";
 import "./Peers.css";
+import { PeersData } from "../redux/actions";
 
-export const Peers = ({ peers }) => {
+type PeersProps = {
+  peers: PeersData[] | null;
+};
+
+export const Peers: FC<PeersProps> = ({ peers }) => {
```
Comment (danielabbasi) [inline src/features/peers/components/Peers.tsx:13]: Q: What is a FC and what characteristics does it take for this to be one?

Comment (zhouyuhe) [reply src/features/peers/components/Peers.tsx:13]: FC is a functional component as I will pass the prop inside.

File: src/features/peers/redux/actions.ts (hunk 1 above)
Comment (zhouyuhe) [reply src/features/peers/redux/actions.ts:6]: will put it into issues

File: src/features/peers/redux/reducer.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { RESET } from "../../../store/constants";
```
Comment (mattydono) [inline src/features/peers/redux/reducer.ts:1]: So in order to remove this deep nesting of imports take a look at `baseUrl` for your tsConfig file.

with `./src` you'll be able to change this 'to store/constants'.

I'd only do this for imports directly from src level as having nested structure can be helpful when navigating through isolated file structures.

**Review by mattydono — APPROVED** [review]

**Pull request merged**